index.html

```diff
@@ -91,30 +91,24 @@ <h3>Răzvan Cîrlugea</h3>
             <div class="custom-card-wrapper col-12 col-md-6 col-lg-4">
               <div class="custom-card">
                 <div class="avatar-wrapper">
-                  <img src="img/placeholder.jpeg" alt="" class="avatar" />
+                  <img src="img/Toteanu-Mihai-Marius-poza.jpg" alt="Toteanu Mihai Marius" class="avatar" />
                 </div>
                 <div>
-                  <h3>Full Name</h3>
+                  <h3>Toteanu Marius</h3>
                   <p>
-                    Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-                    Molestiae omnis nobis quo dicta culpa eum exercitationem
-                    provident ex iure pariatur? Tempora sunt accusamus tempore
-                    modi.
+                    <span class="marius-quote">"At the end of the day, your great achievement is your smile"</span> starting from phrase that leads my life, I would describe myself as the person who never lose his optimism and shares it with all the people around, because, I believe that, even in the most challenging times, there is always a light, this bringing me to my favourite motto <span class="marius-quote">“The word impossible is not in my dictionary.”-Napoleon Bonaparte.</span>
                   </p>
                   <p>
-                    Quidem, corrupti perspiciatis aliquid dicta cum odio
-                    sapiente tempora quas incidunt adipisci alias praesentium et
-                    excepturi error maxime aspernatur sint qui doloremque
-                    distinctio. Ex, unde.
+                    I am thirsty to apply & learn new technologies, at the same time eager improve my knowledge  by asking feedback & searching by myself.
```

Comment:
`eager improve my knowledge` should be `eager to improve my knowledge`

Reply:
`knowledge  by asking` please remove the extra space

Reply:
Hi, Razvan ! Thank you very much for your feedback ! ( one more eye is always better :) ) & sorry if i was intrusive by adding a new line in CSS.

Reply:
You're welcome, Marius! 🙌🏻 There is still an extra space between `my knowledge` and `by asking`, but that's no big deal, so I'll merge the PR 💪